src/charm.py

```diff
@@ -225,29 +238,30 @@ def write_ceph_cli_config(self) -> None:
             "debug mon": "1/5",
             "debug osd": "1/5",
         }
-        config["client"] = {"log file": "/var/log/ceph.log"}
+        config["client"] = {"log file": f"/var/log/ceph/{unit_name}.log"}
 
-        with Path("/etc/ceph/ceph.conf").open("w") as fp:
+        with ceph_config_file().open("w") as fp:
             config.write(fp)
 
     def write_ceph_cli_keyring(self) -> None:
         """Write Ceph CLI keyring file"""
         config = configparser.ConfigParser()
         config[f"client.{self.app.name}"] = {"key": self.key or ""}
-        with Path(f"/etc/ceph/ceph.client.{self.app.name}.keyring").open("w") as fp:
+        with ceph_keyring_file(self.app.name).open("w") as fp:
             config.write(fp)
 
     def configure_ceph_cli(self) -> None:
         """Configure Ceph CLI"""
-        Path("/etc/ceph").mkdir(parents=True, exist_ok=True)
+        ceph_config_dir().mkdir(mode=0o700, parents=True, exist_ok=True)
         self.write_ceph_cli_config()
         self.write_ceph_cli_keyring()
 
     def ceph_cli(self, *args: str, timeout: int = 60) -> str:
         """Run Ceph CLI command"""
-        cmd = ["ceph", "--user", self.app.name] + list(args)
+        cmd = ["/usr/bin/ceph", "--conf", str(ceph_config_file()), "--user", self.app.name, *args]
```

Comment:
```suggestion
        cmd = ["/usr/bin/ceph", "--conf", ceph_config_file().absolute().as_posix(), "--user", self.app.name, *args]
```

Reply:
Great suggestion... i pushed a new commit to encompass that
https://github.com/charmed-kubernetes/ceph-csi-operator/pull/23/commits/70cdf49005757b522013c91e81388c4ff82d26e8